### PR TITLE
New package Microsoft.FSLogix

### PR DESCRIPTION
**Pull request by FaserF**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Is there a linked Issue? #11890

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/165828)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 887.)

**Comment by FaserF** [comment]
I need help here, this looks to me like a bug within winget itself. It downloads the zip file as an exe file (for whatever reason), and then tries to open the zip file as an exe file which results in an error, that this app is not compatible with this device. 
image

But the installer URL clearly is a ZIP file: 
InstallerUrl: https://download.microsoft.com/download/e/c/4/ec4b55b3-d2f3-4610-aebd-56478eb0d582/FSLogix_Apps_2.9.8884.27471.zip

Downloading it manually works completly fine.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 887.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 887.)

**Comment by FaserF** [comment]
@stephengillie maybe this now is the issue for this error? I have now removed the 32bit installation option, thanks as always for your great input!

https://github.com/microsoft/winget-pkgs/pull/165828#issuecomment-2268245325

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 887.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

I'm not sure why this worked on the 64-bit pipeline VM, but isn't working on my VM. At first glance, it appears to be the inexplicable situation of a 64-bit application using an installer that does not work on 64-bit OSes.

**Comment by FaserF** [comment]
Still for me this looks like a winget bug. There is no "FSLogix_Apps_2.9.8884.27471.exe", it is a zip file containing another exe called  "FSLogixAppsSetup.exe".
I dont understand why winget downloads the zip file but renames it to an .exe file. Then you get this error message, since windows trys to open a zip file as an exe file, which of course cannot be read. 
See https://github.com/microsoft/winget-pkgs/pull/165828#issuecomment-2268245325

File: manifests/m/Microsoft/FSLogix/2.9.8884.27471/Microsoft.FSLogix.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.FSLogix
+PackageVersion: 2.9.8884.27471
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/FSLogix/2.9.8884.27471/Microsoft.FSLogix.installer.yaml:6]:
```suggestion
InstallerType: zip
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> I dont understand why winget downloads the zip file but renames it to an .exe file.

Thank you - this reminded me of an uncommon situation. The package manager is doing what it's told - download the file and then "process as EXE". Since it is a ZIP file, we want to change this to "process as ZIP".

File: manifests/m/Microsoft/FSLogix/2.9.8884.27471/Microsoft.FSLogix.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/m/Microsoft/FSLogix/2.9.8884.27471/Microsoft.FSLogix.installer.yaml:6]: I forgot to mention that `zip` requires the `NestedInstaller` to be described:

```
    NestedInstallerType: exe
    NestedInstallerFiles: 
      - RelativeFilePath: FSLogixAppsSetup.exe
```

**Comment by stephengillie** [comment]
Verify:
image

I tried one other application, and it gave an error:
image
Guessing ``frxshell.exe` isn't meant to be run directly.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**